org.eclipse.winery.tools.deployablecomponents/src/main/java/org/eclipse/winery/tools/deployablecomponents/fileanalyzer/CommandAnalyzer/ApkAnalyzer.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+package org.eclipse.winery.tools.deployablecomponents.fileanalyzer.CommandAnalyzer;
+
+import org.eclipse.winery.tools.deployablecomponents.commons.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApkAnalyzer implements CommandAnalyzer {
+    public List<Component> analyze(String command) {
+        if (!command.contains(" add ")) {
+            return new ArrayList<>();
+        }
+        command = command.replace(Commands.Apk.asString(), "");
+        String[] words = command.split("\\s");
+        List<String> packets = new ArrayList<>();
```

Comment:
the correct term here would be "tokens"

Reply:
I disagree. But packets is also wrong. In this context the extracted parts of the analyzed commands are software packages. I rename it to packages in all analyzer-classes.